lib/shared/log/toConsole.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var fancyLog = require('fancy-log');
+
+// The sorting of the levels is
+// significant.
+var levels = [
+  'debug', // Logs all levels.
+  'info',  // Logs info, warn and error.
+  'warn',  // Logs warn and error.
+  'error', // Logs error.
```

Comment:
@kahlil yep, I've updated `fancy-log`. The `log.error()` API is available in 1.1.0 - https://github.com/phated/fancy-log